Runtime/BehaviorCollection.cs

```diff
@@ -25,7 +25,7 @@ public class EntityData : EntityCollectionData<IBehavior>, IBehaviorCollectionDa
             /// List of all <see cref="IBehavior"/>s added.
             /// </summary>
             [DataMember]
-            [DisplayName("Behaviors"), Separated, Foldable, ListOf(typeof(FoldableAttribute), typeof(DeletableAttribute), typeof(SeparatedAttribute), typeof(DrawIsBlockingToggleAttribute)), ExtendableList]
+            [DisplayName(""), ListOf(typeof(FoldableAttribute), typeof(DeletableAttribute), typeof(DrawIsBlockingToggleAttribute)), ExtendableList]
```

Comment:
Why no display name anymore? Asking for a friend

Reply:
because the name is now displayed inside the tab instead :)

Reply:
It duplicates the tab name.